SimpleXML/SimpleXML.php

```diff
@@ -215,6 +215,38 @@ public function __toString () {}
 	 */
 	public function count () {}
 
+    /**
```

Comment:
Methods below are not needed as IDE checks interface only and this can lead to errors in code completion.